DOCS-1079: Update gripper and input controller config instructions (#1978)
viamrobotics/docs: docs/components/gripper/softrobotics.md

![Creation of a softrobotics gripper component in the Viam app config builder.](/components/gripper/softrobotics-ui-config.png)

Copy and paste the following attribute template into your gripper's **Attributes** box.

> Then remove and fill in the attributes as applicable to your gripper, according to the table below.

Reviewer: Nit: Instructions to paste this code then remove this code then fill in this code might confuse (remove suggests 'delete line' not 'delete key's value'). What do you think of "update"?

```suggestion
Then, update the placeholder attribute values as applicable to your gripper, according to the table below.
```

Reviewer: Ahh, makes sense! If some lines are indeed to be removed entirely, then I think the way you have it here works well! Sorry, didn't consider general resource type attributes that don't apply to some constituent models of that type. Retracted!

Author: Or rather, model attributes that do apply to any component of a given model, but that are optional because they either default to something behind the scenes, or don't need to be constrained to a given range unless the user wants them to be, etc.
In other words, it's not that there's a set of attributes for all models of a given type and don't apply to some. Instead, it's that each model has a set of attributes but the user doesn't necessarily need to specify them all, depending on their specific robot and use case.

Reviewer: Gotcha, thanks!
